infrastructure/code/settings.go

```diff
@@ -17,11 +17,68 @@ package code
 
 import (
 	"net/url"
+	"os"
+	"strconv"
 	"strings"
+	"sync"
 
 	"github.com/snyk/snyk-ls/application/config"
+	"github.com/snyk/snyk-ls/internal/concurrency"
 )
 
+const (
+	// String "true" or "false", see `defaultAutofixEnabled`
+	autofixEnabledEnvVarKey = "SNYK_AUTOFIX_ENABLED"
+	// By default this is disabled
+	defaultAutofixEnabled = false
+)
+
+var (
+	codeSettingsSingletone      *codeSettings
```

Comment:
Sorry for being grammar nazi - `Singleton`. Singletone sounds either Italian or French, not sure yet 😄

Reply:
✅